attiny/attiny.ino

```diff
@@ -315,13 +315,22 @@ void setBlinks(int i) {
   }
 }
 
-//========POWER PIN=========
+//========POWER LED PIN=========
 // Slowly turns the LED on and off
 int powerLedState = 1;  // Pulsing on or off
 int powerLedIntensity = 0;
 int blinkTimer = 0;
+bool ledEnabled = true;
 
 void update_power_led() {
+  if (!ledEnabled) {
+    return;
+  }
+  if (millis() > POWER_LED_ON_DURATION) {
```

Comment:
This works because the reset button also reset the attiny right?

Reply:
Yep